wallet2/db/query.go

```diff
@@ -0,0 +1,19 @@
+package db
+
+const (
+	EmptyQuery = ""
+)
+
+type QueryOption string
+
+func WithTransactionStatus() string {
```

Comment:
Maybe like
`func (qo *QueryOption) WithTransactionStatus() *QueryOption {...}`

Reply:
I think this way which we've used is better and I should delete `QueryOption` type. do you agree?